meetings/2021-07-08.md

```diff
@@ -0,0 +1,27 @@
+# ISIS TC Meeting July 8, 2021 @1PM PDT & MST / 4PM EDT
+
+## [Teams Link](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2f_%23%2fl%2fmeetup-join%2f19%3ameeting_YWRkZjdiMGUtZWJlOC00OWMzLThlMTItZTk0Y2MyM2E1MWE0%40thread.v2%2f0%3fcontext%3d%257b%2522Tid%2522%253a%25220693b5ba-4b18-4d7b-9341-f32f400a5494%2522%252c%2522Oid%2522%253a%2522c27c6e98-e45a-45ff-aea5-7f10d6fe67c1%2522%257d%26anon%3dtrue&type=meetup-join&deeplinkId=e54b3969-3c7f-4efb-9cad-ee99cf639f86&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true)
+
+### Attending
+
+-
+
+## Agenda / Notes
+
+- Action Items from last meeting
+
+- Versioning & Breaking Changes OR Testing & Validation
```

Comment:
We need to pick one of these

Reply:
I think to get the LTS model moving, Versioning & Breaking Changes is probably best to tackle in this next meeting.